MMM-WeatherChart.js

```diff
@@ -539,7 +581,9 @@ Module.register("MMM-WeatherChart", {
             y2_max =
                 Math.max(maxRain, maxSnow, this.config.rainMinHeight) *
                 (2 + iconTopMargin + iconBelowMargin + tempRainMargin),
-            y2_min = 0;
+            y2_min = 0,
+            y3_max = maxPressure + 20,
+            y3_min = minPressure - 20;
```

Comment:
y3_max and y3_min should be carefully calculated to make an appropriate margin. If it's difficult, I'll try to fix after the PR is merged.

Reply:
Updated in commit: 90dff89e4954dfea69af4b40754f8bb7c519a05a